cmake/protobuf-module.cmake.in

```diff
@@ -1,7 +1,4 @@
-if(PROTOBUF_SRC_ROOT_FOLDER)
-  message(AUTHOR_WARNING "Variable PROTOBUF_SRC_ROOT_FOLDER defined, but not"
-    " used in CONFIG mode")
-endif()
+# Functions
```

Comment:
A good idea. In fact, I do. Part of the file after determining the functions of strongly altered to determine the relevant variables, and it is a big part of the file. Simply include the module will not work - you need to check work for all supported versions of CMake. The module had a change over time.